sjsonnet/server/src/sjsonnet/SjsonnetServerMain.scala

```diff
@@ -24,7 +24,11 @@ trait SjsonnetServerMain[T]{
             wd: os.Path): (Boolean, Option[T])
 }
 
-object SjsonnetServerMain extends SjsonnetServerMain[collection.mutable.HashMap[(Path, String), Either[String, (Expr, FileScope)]]]{
+//  [Ask] SjsonnetServerMain has a valid main method (args0: Array[String]): Unit,
+//  [warn]   but sjsonnet.SjsonnetServerMain will not have an entry point on the JVM.
+//  [warn]   Reason: companion is a trait, which means no static forwarder can be generated.
+//  [warn] object SjsonnetServerMain extends SjsonnetServerMain[DefaultParseCache]{
```

Comment:
What's with this comment? Is this an issue that should be addressed?

Reply:
This got generated by I ran `./mill __.test`, I am not sure why this came but after some changes this doesn't come however the comment already present - "Disable SIGINT interrupt signal in the Mill server." got more detailed.